RetailSuiteWarehousing/Integration/02_ConfigureWarehouses.md

```diff
@@ -0,0 +1,118 @@
+[!!Configure the warehouse groups](./01_ConfigureWarehouseGroups.md)
+[!!Check the posting history](../Operation/03_CheckPostingHistory.md)
+[!!User interface Warehouse](../UserInterface/03a_Warehouse.md)
+
+# Configure the warehouses
+
+A warehouse is a subordinate storage area within a warehouse group used for a specific purpose, such as sales, returns, or blocked stock. A warehouse can be assigned, or not, to a specific warehouse group. If a warehouse is not assigned to any warehouse group, it will be listed under the general *No warehouse group* in the left side bar. 
```

Comment:
Vorschlag: A warehouse is a subordinate storage area within a warehouse group. It is used for a specific purpose, such as sales, returns, or blocked stock. A warehouse may or may not be assigned to a specific warehouse group. If a warehouse is not assigned to any warehouse group, it will be listed under the general *No warehouse group* in the left side bar.